### PR TITLE
Ingest Crystallographic Info

### PR DESCRIPTION
**Pull request by rboston628**
See EWM 774

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -20,6 +20,7 @@ dependencies:
 - setuptools
 - python-build
 - boa
+- pycifrw
```
Comment (walshmm) [inline environment.yml:23]: where is this used? ctrl+f doesnt reveal anything

Comment (rboston628) [reply environment.yml:23]: This is needed in order for LoadCIF to work.  See output on failing test here: https://github.com/neutrons/SNAPRed/actions/runs/4995107184/jobs/8946555614#step:6:161

Comment (walshmm) [reply environment.yml:23]: sounds good

File: src/snapred/backend/service/CrystallographicInfoService.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+from typing import Any, Dict
+
+from snapred.backend.data.DataFactoryService import DataFactoryService
+from snapred.backend.recipe.CrystallographicInfoRecipe import CrystallographicInfoRecipe
+from snapred.meta.Singleton import Singleton
+
+
+@Singleton
+class IngestionService:
```
Comment (walshmm) [inline src/snapred/backend/service/CrystallographicInfoService.py:9]: Could you refactor this to be in line with the new way we do services?
i.e. rename the orchestrate method, decorate it with FromString, and register the former orchestrate method in init to "", and have IngestionService inherit from Service

File: tests/unit/backend/service/test_ExtractionService.py
Comment (rboston628) [inline tests/unit/backend/service/test_ExtractionService.py:1]: This test didn't do anything, so I removed it

**Pull request merged**